### ISSUE TITLE
Getting NullPointerException while connecting to another Wi-Fi Direct device

### ISSUE THREAD

**Issue by rahulfoa** [deib-polimi/PigeonMessanger#2]
I tried to connect to another device logcat follows the following exception (attached).

Please help me resolve the issue.

![exception](https://cloud.githubusercontent.com/assets/4984274/14768842/6f3f8ef8-0a69-11e6-86ec-273007173a10.JPG)

The devices were discovering each other and the "GO" icon came. As soon as I selected the device the application crashed with the aforementioned exception.

Devices used: 
Yu Yuphoria (CM 12.1) Android  5.1.1 API 22
Lenovo A7000 API 21

**Comment by Ks89** [comment]
Issue closed.
Check the new release here (with a small changelog): https://github.com/deib-polimi/PigeonMessanger/releases/tag/v.alpha-3

If you have questions, I'm here :)

PS: I decided to revert some code. Also I found a problem with the latest support libraries (recyclerview/cardview). I don't know if it's a bug or my fault, but with the master branch and this compiled version you'll be able to run Pigeon Messenger (tested on Android 6.0.1 and also Android N dev preview 2).

Thank you very much for your interest.
